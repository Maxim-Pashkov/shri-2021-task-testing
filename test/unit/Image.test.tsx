import { it, expect, describe } from '@jest/globals';
import { render } from '@testing-library/react';
import { Image } from '../../src/client/components/Image';


describe('Image test', () => {
    it('Image outputed', () => {
        const application = <Image />;

        const { queryByRole } = render(application);

        expect(
            queryByRole('img')?.hasAttribute('src')
        ).toBeTruthy();
    });
});
