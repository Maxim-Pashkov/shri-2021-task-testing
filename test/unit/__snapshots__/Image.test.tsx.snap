// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image test Image outputed 1`] = `
<img
  class="Image"
  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkMAYAADkANVKH3ScAAAAASUVORK5CYII="
/>
`;
